_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,371 @@
+  ([transcript][socratic ts], [video][socratic vid])
+
+    The following day, Tim Ruffing presented on multisignature and
+    threshold signature schemes using schnorr signatures. He highlighted
+    how the rogue key attack and an attack using Wagner’s algorithm (see
+    [Newsletter #74][news74 wagner]) need to be addressed when designing
+    secure multisignature schemes.  Ruffing has been working with
+    collaborators on a speculative [MuSig][topic musig] signature scheme
+    that only requires two rounds of interaction rather than three
+    (without relying on zero knowledge proofs). Threshold signature
+    schemes generally present further challenges such as requiring an
+    honest majority, a broadcast channel, and careful consideration of
+    parallel signing sessions.  ([transcript][ruffing ts],
+    [video][ruffing vid], [slides][ruffing slides])
+
+
+- **Sydney meetup discussion:** A number of Bitcoin developers and LN
+  developers joined this Sydney meetup to discuss various topics. Ruben
+  Somsen gave a short presentation on Succinct Atomic Swaps (SAS), see
+  [Newsletter #98][news98 sas].  Somsen explained how SAS compared to
+  Chris Belcher’s CoinSwap proposal, whether atomic swaps via privacy
+  focused altcoins is a workable approach to obtain additional privacy,
+  and how SAS only requires one of the two block chains to have timelocks
+  and scripting opcodes.
+
+    Other non-related topics included reliance on DNS seeds in Bitcoin
+    Core, whether they present a viable attack vector against new full
+    nodes seeking to locate network peers for the very first time, and
+    how work by Matt Corallo and Antoine Riard on downloading block headers
+    over HTTP and allowing other [alternative transports][Bitcoin Core
+    #18988] could potentially mitigate this risk. Lloyd Fournier
+    also discussed how experimenting with his toy Rust implementation of
+    secp256k1 (secp256kfun) led to a small [fix][libsecp256k1 #732] in the ECDSA signature
+    code in the actual secp256k1 library. The transcript was anonymized
+    to encourage open discussion.  ([transcript][sydney ts])
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 1.1.2-rc.2][hwi 1.1.2]: this release candidate includes support
+  for handling [PSBTs][topic psbt] with serialized previous segwit
+  transactions, which is now required or suggested for several hardware
+  wallets in response to the [fee overpayment attack][news101 fee
+  overpayment].  Experienced users, and developers using HWI in their own
+  applications, may wish to test whether this prerelease resolves the
+  issues related to recent hardware wallet firmware updates.
+
+- [LND 0.10.2-beta.rc2][lnd 0.10.2-beta]: this release candidate for an
+  LND maintenance release is now available for testing.
```

Comment:
One project to consider covering in terms of releases might be JoinMarket. Mentioning it because waxwing is announcing a release: https://x0f.org/@waxwing/104428846061585630

Reply:
I'd love to cover JoinMarket, but I think it'd be hard for us to start covering it without also covering Wasabi and maybe Samourai Wallet out of fairness.  Although it's not a hard rule, currently I try to only use this section and the notable merges section for infrastructure projects---software that's often used in other software or services.

Reply:
I agree, infra is a good distinction/place to draw the line, and it's good to say it here in the intro to this section. I wonder if it should be clarified in the opening summary where the term is already present.